components/sandbox/index.js

```diff
@@ -142,6 +142,8 @@ export default class Sandbox extends Component {
 			}
 		`;
 
+		const wpEmbedScript = 'wp-embed' === this.props.type && window._wpEmbedScript ? ( <script type="text/javascript" src={ window._wpEmbedScript }></script> ) : '';
```

Comment:
Rather than making the Sandbox component aware of _types of scripts_, I wonder if it should instead accept a `children` prop that allows additional elements to be injected into the sandboxed content.

Reply:
If there were such a property, i guess it should be an object with at least two keys eg `{ head: 'to add styles', footer: 'to add JavaScripts' }` as so far there's only some specific code for Video Embeds.

Btw, i've cleaned up the mess i put when i rebased earlier.

Reply:
>If there were such a property, i guess it should be an object with at least two keys

Perhaps. Though, to the best of my knowledge, loading a stylesheet at the end of a page is totally valid.

https://stackoverflow.com/a/21749882/995445

Alternatively, assuming appending styles and scripts are the common use case, we could specify `styles` and `scripts` as array props of URL strings, e.g.:

```
<Sandbox scripts={ [ window._wpEmbedScript ] } />
```

Related prior art: https://github.com/Automattic/wp-calypso/tree/master/client/lib/embed-frame-markup

>If there were such a property

We can/should introduce it here, since it's as part of the introduction of the requirement to include an additional script in the sandbox code.